### PR TITLE
patch: update readme

### PR DESCRIPTION
**Pull request by NajeebTyson**

### CODE REVIEW

**Review by zubair-nabi — COMMENTED** [review]
Should we update the README to reflect that we now have two ways of crawling events: 1) Cloudtrail -> SQS/S3, and 2) Organizational Cloudtrail -> Eventbridge?

File: README.md [hunk 1]
```diff
@@ -57,9 +57,14 @@ To use these Terraform scripts, ensure that you have the following prerequisites
 2. Navigate to the respective module directory (`azure` or `aws`).
 3. Configure the required variables and provider settings in the `variables.tf` and `provider.tf` files.
 4. Run `terraform init` to initialize the working directory.
-5. Run `terraform plan` to review the planned infrastructure changes.
-6. Run `terraform apply` to apply the changes and provision the resources.
-7. After successful provisioning, the outputs will be displayed. Make note of the relevant information for further use.
+5. For the next commands its mandatory to pass the S3 bucket name as a variable for storing SQS logs. To get the bucket name perform the following steps:
```
[the remark this answers is not in the document]
Comment (NajeebTyson) [reply README.md:60]: yes we create this through terraform. And yes we don't have default value for this so it asks for it when we run terraform. I think we can also make up its value based on the env. I am not sure why its not like this already

File: README.md [hunk 2]
```diff
@@ -29,7 +29,23 @@ The following outputs are provided:
...
-The AWS module provisions AWS resources using Terraform. It creates the following resources:
+The AWS module provisions AWS resources using Terraform. 
+
+### Event Crawling Methods
+
+Our system supports two distinct methods for crawling events, allowing flexibility based on your AWS environment and requirements.
+
+### 1. CloudTrail to SQS/S3
+
+In this method, events are ingested from AWS CloudTrail. The events are either delivered AWS S3 bucket through an Amazon Simple Queue Service (SQS). This approach provides a scalable and durable solution for capturing events.
```
Comment (zubair-nabi) [inline README.md:40]: We should also say that this applies to AWS accounts that do not have an organizational level CloudTrail enabled and hence a custom CrawlTrail needs to be created.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (zubair-nabi) [reply README.md:76]: Yeah, I don't think we should get this from the user. We should just standardize the name and parameterize based on the env.

**Review by zubair-nabi — COMMENTED** [review]
Added a couple of comments.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (NajeebTyson) [reply README.md:76]: Ok I will create a ticket for this.

File: README.md (hunk 2 above)
Comment (NajeebTyson) [reply README.md]: done

**Review by zubair-nabi — APPROVED** [review]
LGTM!

**Review by muhammad-tahir-nawaz — APPROVED** [review]
LGTM!

**Pull request merged**